### PR TITLE
fixed itemCounter, fixed startUrl for non-english versions

### PR DESCRIPTION
**Pull request by OlehVeselov92**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -81,13 +85,12 @@ Apify.main(async () => {
...
             if (!req.userData) req.userData = {};
             if (!req.userData.label) req.userData.label = 'START';
-            req.userData.itemsCounter = itemsCounter;
             req.userData.currentPageNumber = currentPageNumber;
             if (req.url.includes("viewjob")) req.userData.label = 'DETAIL'
             if (!req.url.includes('&sort=date')) req.url = `${req.url}&sort=date` // with sort by date there is less duplicates in LISTING
             await requestQueue.addRequest(req);
             log.info(`This url will be scraped: ${req.url}`);
-            countryUrl = req.url.split('.com')[0] + '.com';
+            countryUrl = `https://${req.url.split('https://')[1].split('/')[0]}`
```
Comment (metalwarrior665) [inline src/main.js:93]: It is kinda weird that it is applied in this loop but to the top level. Seems that doesn't handle that there can be multiple countries in the URLs. Why do we need the countryUrl on the top level and not on the request?

**Review by metalwarrior665 — CHANGES_REQUESTED** [review]

File: src/main.js (hunk 1 above)
Comment (OlehVeselov92) [reply src/main.js:93]: @metalwarrior665 
It was originally like this.
So I decided to leave as it is.
It works correct. I tested it on 7 different countries.

And actually, as I see, we use it on a top level for case if there is no START URL in input, and we have to build it on our own with the query and location from input.

Comment (metalwarrior665) [reply src/main.js:93]: I don't want to dig and block the update but would be good to refactor this part later if this is a noise

**Review by metalwarrior665 — APPROVED** [review]

**Comment by OlehVeselov92** [comment]
@metalwarrior665 
ok, thanks.
During next update, I will keep this case in mind.

**Pull request merged**